src/__tests__/Guest.test.tsx

```diff
@@ -1,22 +1,47 @@
 import React from 'react';
 import { render, screen, waitFor } from '@testing-library/react';
 import { MemoryRouter } from 'react-router-dom';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event'
 
 import Guest from '../components/Guest/Guest';
 
 describe('Guest', () => {
   it('renders Guest card', () => {
+    const deleteGuest= jest.fn()
```

Comment:
This function might need to change to async once we get the api for delete going